astropy/coordinates/tests/test_geodetic_representations.py

```diff
@@ -24,6 +27,18 @@
 from astropy.tests.helper import assert_quantity_allclose
 from astropy.units.tests.test_quantity_erfa_ufuncs import vvd
 
+# Preserve the original REPRESENTATION_CLASSES dict so that importing
+# the test file doesn't add a persistent test subclass (CustomGeodetic, etc.)
+REPRESENTATION_CLASSES_ORIG = deepcopy(REPRESENTATION_CLASSES)
```

Comment:
Should this be in `setup_module` to be consistent with teardown?

Reply:
I started that, but then I still need to define global variables, no? In the end, it seemed an unnecessary complication.  But maybe I was overcomplicating things - how should this be done?

Reply:
Hmm, good point. I guess this is fine since pyinstaller is happy. Thanks for the clarification!